### ISSUE TITLE
Criteria for including other implementations

### ISSUE THREAD

**Issue by jpellegrini** [schemedoc/srfi-metadata#35]
What would be the criteria for listing Scheme implementations on the table?
I see that Biwa and Cyclone are missing, and these are actively maintained; and there are others in the table that don't seem to be updated for years.

**Comment by lassik** [comment]
There are no particular criteria AFAIK; people just added what they needed. Cyclone and Biwa should be added, but the table will get quite hard to read with more implementations. We should think of some criteria for what to show by default, and some user interface for showing and hiding particular implementations from the table. Maybe checkboxes / buttons?

**Comment by jpellegrini** [comment]
I agree with having checkboxes for choosing implementations. As to default criteria, recent development activity is the only objective criterion I can think of.

**Comment by lassik** [comment]
I like that idea since it rewards implementations for their work.

**Comment by jpellegrini** [comment]
That means Scheme48 and Ypsilon wouldn't be listed by default. I think the Scheme48 authors mentioned that they won't be working on it anymore. And Ypsilon hasn't been updated in several years.

**Comment by lassik** [comment]
I agree that Scheme48 and Ypsilon are more of historical interest nowadays. Though still inspiring and usable.

**Comment by jpellegrini** [comment]
> I agree that Scheme48 and Ypsilon are more of historical interest nowadays. Though still inspiring and usable.

Yes - I do think they should be listed, but maybe not by default... It's a SRFi-support table anyway.  One who's looking at that table is likely deciding which implementation to use for a project, and I wouldn't recommend either of those two, not because they're not good - they're both excellent - but because they're not actively maintained.

**Comment by jpellegrini** [comment]
> for a project

Or to start learning; or to pick SRFI code and see how it's implemented in different Schemes (I do that). But I guess having Biwa and Cyclone would be more important, although I'm not really sure. 

What do you think?

**Comment by lassik** [comment]
It would probably be most useful to list actively maintained implementations geared toward writing practical programs first.

**Comment by jpellegrini** [comment]
Ok, so maybe not Biwa, but Cyclone?
Although Biwa seems quite easy to use for a beginner looking for a simple REPL.

**Comment by lassik** [comment]
I don't have an opinion on that. Ideally we would have some kind of objective criteria, but we don't yet.

**Comment by jpellegrini** [comment]
If all implementations had public git (or any version control) access, it would be easy to come up with a "commit volume" creterion.
But some (like Tinyscheme) don't... So an objective criteria is really hard to define.  Having released a version  in the last 3 years?

**Comment by lassik** [comment]
True. Date of latest release is one criterion. But if we sort by some kind of date, the order would change quite often.

Unless we filter by date, and those implementations that pass the filter (e.g. top 10 by date), we sort alphabetically.

**Comment by jpellegrini** [comment]
> Unless we filter by date, and those implementations that pass the filter (e.g. top 10 by date), we sort alphabetically.

That's what I meant, but I think I didn't make it clear... :)

**Comment by jpellegrini** [comment]
Vicare is also not being developed anymore (it's stated in their webpage).
